hiring/README.md

```diff
@@ -0,0 +1,41 @@
+# Our hiring process
+
+We believe that our hiring process should be [fair and repeatable](rationale.md).
+
+## Pipeline
+
+ - Defining a need and expectation for hiring
+ - Screening
+ - Interview & Pairing
+ - Hire
+ - Nurture
+
+Our pipeline is tracked on a [trello board](https://trello.com/b/0kmnv5Wo/recruitment).
+
+### Defining a need and expectation for hiring
+
+As described in our [Roles at Made Tech](../roles/README.md) document, candidates must be judged against a specific role with the needs and expectations of that role explicitly defined. This allows us to judge candidates like for like and also keeps us focussed on the reason for a hire.
+
+### Screening
+
+Our initial screening stage is usually a phone interview for a brief conversation to see where a candidate is at in their careers, talk about Made and perhaps have a technical chat about past experiences.
+
+Without clear examples of engineering competency a candidate may be asked to complete our [remote test](remote_test/README.md).
+
+### Interview & Pairing
+
+The candidate is invited into our offices for an hour and a half. Usually the first 30 minutes will be spent talking more in depth about the candidates previous experience of processes and practices along with a more detailed explanation of how we work.
+
+After this introduction we usually pair with a candidate on a problem for 30 minutes. This step is used for seeing how well a candidate communicates and works with us. Of course it is a pressured environment, however we try and make the candidate at ease by sharing some of the work and discussing the problem with them.
```

Comment:
Do we need to define some of the more practical things. e.g. Bring a laptop to your interview, have a development environment setup, let us know what language you want to pair in, etc? I find myself constantly writing up a list of things and sharing with candidates pre-interview. Might be useful to get this in here.